### PR TITLE
C now links to composer

### PR DESCRIPTION
**Pull request by clloyd**
This adds a link back to composer from the "C" link using the /find-by-path/ functionality in composer

### CODE REVIEW

File: app/com/gu/viewer/config/Configuration.scala [hunk 1]
```diff
@@ -13,15 +13,13 @@ object Configuration {
...
     case Some(value) => value
     case None => "DEV" // default to DEV stage
   }
 
-
   val previewHost = getConfigString(s"previewHost.$stage")
   val liveHost = getConfigString(s"liveHost.$stage")
   val mixpanel = getConfigString(s"mixpanel.$stage")
-
+  val composerReturn = getConfigString(s"composerReturnUri.$stage")
```
Comment (rrees) [inline app/com/gu/viewer/config/Configuration.scala]: Why lookup config here instead of just templating a string by stage?

**Comment by rrees** [comment]
:shipit:

File: app/com/gu/viewer/views/viewer.scala.html [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+@(viewerUrl: String, previewEnv: String, composerUrl: String)
 
 @main("Viewer") {
     <header class="fixed-header">
         <div class="top-toolbar">
             <div class="top-toolbar__title">
-                <div class="top-toolbar__letter">
-                    <span class="top-toolbar__letter-text">C</span>
-                </div>
+                <a href="@composerUrl">
```
Comment (jamespamplin) [inline app/com/gu/viewer/views/viewer.scala.html]: Can we please pop a `title` tag on this, some like "View in Composer".

**Comment by jamespamplin** [comment]
:+1: 1 small comment on title text, but otherwise looks good

**Pull request merged**